tutorials/ph-mult-rasters/index.md

```diff
@@ -0,0 +1,131 @@
+
+In this step, we are going to perform the following tasks in sequence as part of a function called "GetImage":
+
+1. We first have to tell Google Earth Engine that the dates we have defined above are in fact date variables.
+2. Next, we filter on the start and end dates specified earlier, and the region object we defined by drawing a polygon around Botswana
+3. It's good practice to check how many images we get - we want to just keep one
+4. We take the earlier image by sorting on this element in the data
+5. Our function spits out a singe image in Botswana for the season we specify
+
+```
+
+// Write a function to grab image with data you need
+var getImage = function(bdt, edt) {
+  
+var start = ee.Date(bdt); // this is converting the inputted element from stArray to date
+var end = ee.Date(edt); // this is converting the inputted element from eArray to date
+
+// Apply filter to a set of images
+var lanset = ee.ImageCollection("NOAA/DMSP-OLS/NIGHTTIME_LIGHTS")
+  .filterDate(start,end) // search within start, end dates
+  .filterBounds(region); // define polygon (can draw or import)
+
+
+  //How many images?
+  print(lanset, "print lanset"); // this tells us the type of object we are looking at
+  var count = lanset.size(); // this tells us how many images meet criteria
+  print(count); // print shows us output
+  
+  // take earliest
+  var recent = ee.Image(lanset.sort('system:time_start').first()); 
+  // this code is important. We are going from "ImageCollection" to "Image" using
+  // ee.Image, and then we apply a 'sort' function on 'system:time_start' to get 
+  // earliest image
+
+  print(recent, "print recent"); // sanity check to make sure we have the image
+  return recent; // we output the image we have selected
+};
+
+```
+## Step 4: Export our rasters to Google Drive
+
+In this code, we are combining the functions we defined earlier to produce a set of seasonal images for each year. In this case, because we have only defined a single season (January), we will end up with 3 raster image files (for 01-2000, 01-2001, 01-2002). We perform the following steps:
+
+1. We create counters for year, and season, allowing us to iterate over the elements in those arrays
+2. We pull our image using the "GetImage" function
+3. We pull the bands that we want to use for our analysis - in this case, it is the "avg_vis" variable with average land surface temperature
+4. We then use the "image.toDrive" command to export the raster to our Google Drive. Our loops allow this to happen in sequence, and then we can hit "RUN" in the tasks window to download the rasters (note the scale: 1000, and crs before pulling into another package to link with spatial health data).
+
+```
+// use loops over years, start, end dates to pull images
+
+var i; // year counter
+for (i = 0; i < yrarr.length; i++) { // years
+    var j; // MM-DD counter
+    for (j=0; j < 1; j++){ // change for number of elements in stArr
```

Comment:
@jdbcode Is this the right way to export multiple images?

Reply:
It exports one at a time, looping over year and then season.. if you guys have a way to do it all at once without loops I am all ears